docs/release/3.3/release-3-3-30-3.md

```diff
@@ -0,0 +1,82 @@
+
+Summary of changes
+------------------
+
+This release contains:
+
+-   CA Certificates based on [IGTF 1.88](http://dist.eugridpma.info/distribution/igtf/current/CHANGES)
+    - updated UKeScience 2B ICA based on a SHA-2 family digest (UK)
+    - added new PKIUNAMgrid (2017) trust anchor for roll-over (MX)
+
+These [JIRA tickets](https://jira.opensciencegrid.org/issues/?jql=project%20%3D%20SOFTWARE%20AND%20fixVersion%20%3D%203.3.30-3%20ORDER%20BY%20priority%20DESC%2C%20key%20DESC) were addressed in this release.
+
+Detailed changes are below. All of the documentation can be found [here](../../).
+
+Updating to the new release
+---------------------------
+
+### Update Repositories
+
+To update to this series, you need [install the current OSG repositories](../../common/yum#install-osg-repositories).
+
+### Update Software
+
+Once the repositories are installed, you can update to this new release with:
+
+``` console
+root@host # yum update
+```
+
+!!! note
+    Please be aware that running `yum update` may also update other RPMs. You can exclude packages from being updated using the `--exclude=[package-name or glob]` option for the `yum` command.
+
+!!! note
+    Watch the yum update carefully for any messages about a `.rpmnew` file being created. That means that a configuration file had been editted, and a new default version was to be installed. In that case, RPM does not overwrite the editted configuration file but instead installs the new version with a `.rpmnew` extension. You will need to merge any edits that have made into the `.rpmnew` file and then move the merged version into place (that is, without the `.rpmnew` extension). Watch especially for `/etc/lcmaps.db`, which every site is expected to edit.
+
+Need help?
+----------
+
+Do you need help with this release? [Contact us for help](../../common/help).
+
+Detailed changes in this release
+--------------------------------
+
+### Packages
+
+We added or updated the following packages to the production OSG yum repository. Note that in some cases, there are multiple RPMs for each package. You can click on any given package to see the set of RPMs or see the complete list below.
+
+#### Enterprise Linux 6
+
+-   [igtf-ca-certs-1.88-1.osg33.el6](https://koji.chtc.wisc.edu/koji/search?match=glob&type=build&terms=igtf-ca-certs-1.88-1.osg33.el6)
+-   [osg-ca-certs-1.68-1.osg33.el6](https://koji.chtc.wisc.edu/koji/search?match=glob&type=build&terms=osg-ca-certs-1.68-1.osg33.el6)
+
+#### Enterprise Linux 7
+
+-   [igtf-ca-certs-1.88-1.osg33.el7](https://koji.chtc.wisc.edu/koji/search?match=glob&type=build&terms=igtf-ca-certs-1.88-1.osg33.el7)
+-   [osg-ca-certs-1.68-1.osg33.el7](https://koji.chtc.wisc.edu/koji/search?match=glob&type=build&terms=osg-ca-certs-1.68-1.osg33.el7)
+
+### RPMs
+
+If you wish to manually update your system, you can run yum update against the following packages:
+
+    igtf-ca-certs osg-ca-certs osg-gums-config vo-client vo-client-edgmkgridmap vo-client-lcmaps-voms
```

Comment:
There isn't a VO update is there? Same goes for the other doc

Reply:
Good catch. I pushed an update with the requested changes.

Reply:
Still missing a change. Ugh.